src/components/PatchActionButtons/UnschedulePatchTasks.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+import styled from "@emotion/styled";
+import { useBannerDispatchContext } from "context/banners";
+import {
+  UnschedulePatchTasksMutation,
+  UnschedulePatchTasksMutationVariables,
+} from "gql/generated/types";
+import { UNSCHEDULE_PATCH_TASKS } from "gql/mutations";
+import { useMutation } from "@apollo/react-hooks";
+import { Popconfirm } from "antd";
+import Checkbox from "@leafygreen-ui/checkbox";
+import { Disclaimer, Body } from "@leafygreen-ui/typography";
+import { DropdownItem } from "components/ButtonDropdown";
+
+interface UnscheduleProps {
+  patchId: string;
+  hideMenu: (e?: React.MouseEvent<HTMLElement, MouseEvent>) => void;
+  refetchQueries: string[];
+  setParentLoading?: (loading: boolean) => void; // used to toggle loading state of parent
+  disabled: boolean;
+}
+export const UnschedulePatchTasks: React.FC<UnscheduleProps> = ({
+  patchId,
+  hideMenu,
+  refetchQueries,
+  disabled,
+  setParentLoading = () => undefined,
+}) => {
+  const { successBanner, errorBanner } = useBannerDispatchContext();
+  const [abort, setAbort] = useState(false);
+  const [
+    unschedulePatchTasks,
+    { loading: loadingUnschedulePatchTasks },
+  ] = useMutation<
+    UnschedulePatchTasksMutation,
+    UnschedulePatchTasksMutationVariables
+  >(UNSCHEDULE_PATCH_TASKS, {
+    onCompleted: () => {
+      successBanner(
+        `All tasks were unscheduled ${
+          abort ? "and tasks that already started were aborted" : ""
+        }`
+      );
+      setAbort(false);
+      hideMenu();
+      setParentLoading(false);
+    },
+    onError: (err) => {
+      errorBanner(`Error unscheduling tasks: ${err.message}`);
+      setParentLoading(false);
+    },
+    refetchQueries,
+  });
+
+  return (
+    <Popconfirm
+      icon={null}
+      placement="left"
+      title={
+        <>
+          <StyledBody>Unschedule all tasks?</StyledBody>
+          <Checkbox
```

Comment:
the label and checkbox look unaligned
<img width="306" alt="Screen Shot 2020-05-26 at 2 46 09 PM" src="https://user-images.githubusercontent.com/10734386/82938647-22ef3400-9f60-11ea-814b-cd7142ea7d84.png">
you may be able to pass jsx into the label prop to fix this

Reply:
<img width="292" alt="Screen Shot 2020-05-26 at 5 44 45 PM" src="https://user-images.githubusercontent.com/15262143/82953369-a74db100-9f78-11ea-89eb-4722a2d993b3.png">

Weird, there's no disalignment on mine.